src/ChaiBuffer.hpp

```diff
@@ -49,14 +48,18 @@ static std::mutex chaiLock;
  */
 inline std::string calculateSize( size_t const bytes )
 {
-  if (bytes >> 20 != 0)
+  std::string size;
+  if( bytes >> 20 != 0 )
   {
-    return std::to_string(bytes >> 20) + "MB";
+    size = std::to_string( bytes >> 20 ) + "MB";
   }
   else
   {
-    return std::to_string(bytes >> 10) + "KB";
+    size = std::to_string( bytes >> 10 ) + "KB";
   }
+
+  // return a space padded size
+  return std::string( 7 - size.length(), ' ' ) + size;
```

Comment:
huh? why 7?

Reply:
That gives you five digit characters in the `XXXXXMB` string where X is either a digit or leading space. That lets you get up to transfers of 99999MB or 97GB which I think is a reasonable upper limit for now.